integration/integration_test.go

```diff
@@ -454,6 +456,83 @@ func TestRelativePaths(t *testing.T) {
 	})
 }
 
+func TestLintDockerfile(t *testing.T) {
+	_, ex, _, _ := runtime.Caller(0)
+	cwd := filepath.Dir(ex)
+
+	testDir := "test_dir"
+	testDirLongPath := filepath.Join(cwd, testDir)
+
+	if err := os.MkdirAll(testDirLongPath, 0750); err != nil {
+		t.Errorf("Failed to create dir_where_to_extract: %v", err)
+	}
+
+	lintOKDockerfile := "Dockerfile_hadolint_OK"
+	lintKODockerfile := "Dockerfile_hadolint_KO"
+	lintSpecialDockerfile := "Dockerfile_hadolint_Special"
+
+	files := map[string]string{
+		lintOKDockerfile:      "FROM debian:9.11\nRUN echo \"hey\"",
+		lintKODockerfile:      "FROM debian\nRUN echo \"hey\"",
```

Comment:
what does `lintKODockerfile` mean? KnockOut was what the only thing I could think of. Preferably change to `lintErrorDockerFile`  unless it's obvious to everyone.

Reply:
Yeah you're right, I will rename this as `lint_DL3006_Dockerfile`, it's even more detailed this way